doc/source/userguide.rst

```diff
@@ -114,15 +114,34 @@ size:
 
 ...
 
-Switching from CPU to GPU
-=========================
+Frontends
+=========
+
+You can choose a frontend via the `frontend=` arguments available in all the Scattering constructors. By default, `pytorch`
+frontend is used.
+
+NumPy
+-----
+
+To call this frontend, simply do::
+
+    import numpy as np
+    from kymatio import Scattering2D
+
+    scattering = Scattering2D(J=2, shape=(32, 32), frontend='numpy')
+
+This will only use standard NumPy routines to calculate the scattering
+transform.
+
+PyTorch
+-------
 
 When initialized, a scattering transform object is stored on the CPU::
 
     import torch
```

Comment:
Needed?

Reply:
Also seems like its needed in this example as we use torch functions later on.

Reply:
Yeah- I think it's a good practice, even for us, to explicitely `import torch` .. I find it clearer

Reply:
> Also seems like its needed in this example as we use torch functions later on.

Ok. Didn't see that. Makes sense.